.github/workflows/test.yml

```diff
@@ -24,28 +27,25 @@ jobs:
         uses: gradle/gradle-build-action@v2.6.0
 
       - name: Execute Gradle build
-        run: ./gradlew clean platform:test
+        run: ./gradlew clean test
 
-      - name: Report
```

Comment:
@ethanmills  I think you added this test reporter, is it better/different to the mikepenz one?

Reply:
he's happy for us to try this